libs/ngx-charts-on-fhir/src/lib/range-selector/range-selector.component.spec.ts

```diff
@@ -59,28 +60,28 @@ describe('RangeSelectorComponent', () => {
   it('should calculate proper 1 month ago date from max layer date', async () => {
     let ButtonInputGroup = await loader.getHarness(MatButtonToggleHarness.with({ selector: "[id='1 mo']" }));
     await ButtonInputGroup.check();
-    const expectedMinDate = new Date('2022-02-28T00:00');
+    const expectedMinDate = subtractMonths(component.maxDate!, 1);
```

Comment:
> 4. Do not use an output of real code as the expected value in an assertion. It might be incorrect.

These test cases are supposed to check whether the code is calculating the moth difference correctly. If you use that exact same function to compute the expected value in the test code, you're not really testing anything. It's like checking whether X = X